src/pds/registry/utils/titan_treks/product_service_builder.py

```diff
@@ -0,0 +1,250 @@
+"""Utility for create_titan_treks.py to build pds4 xml."""
+import xml.etree.ElementTree as Et
+from datetime import date
+
+#
+# This code is very specific to Titan Treks (ie hardcoded urls),
+# but should be able to be generalized easily to other Treks APIs.
+# Currently just a proof of concept for a tool
+#
+
+
+def create_pds4_xml(data, save_xml=False, dest="xml_files", verbose=False):
+    """Creates the pds4 xml labels for the given data.
+
+    :param data: data to fill the labels with
```

Comment:
That would be useful to tell where the data should come from, what its format is (as shortly as possible).